docs/devupdate.md

```diff
@@ -9,6 +9,33 @@ tags:
 
 This page highlights the important changes that are coming in Moodle 4.4 for developers.
 
+## Multiple enrol instances of same type in csv course upload
+
+<Since version="4.3" issueNumber="MDL-79064" />
+
+It is now possible to upload a CSV file with multiple enrol instances of the same type in same course. This is useful for example when you want to enrol users in a course using two different cohorts.
+
+:::caution Format of the CSV file
+
+Please use **only** single line per enrol instance format:
+
+```php
+shortname,fullname,category_idnumber,enrolment_1,enrolment_1_role,enrolment_1_cohortidnumber
+C1,Course 1,CAT1,cohort,student,CV1
+C1,Course 1,CAT1,cohort,teacher,CV4
+```
+
+If a single line format is used, only last enrol instance will be updated. For example
+
+```php
+shortname,fullname,category,summary,enrolment_1,enrolment_1_role,enrolment_2,enrolment_2_role
+shortname,fullname,category,summary,cohort,student,cohort,teacher
+```
+
+will only update the second enrol instance.
+
+:::
+
```

Comment:
Apart from adding this information, I was also expecting to mention the new method, *find_instance* that third-party plugins will need to implement to support this feature.
You can add a sentence and then link it to the section you've created in apis/plugintypes/enrol page :-)